lib/Lemplate.pm

```diff
@@ -538,6 +534,39 @@ local function stash_set(stash, k, v, default)
     end
 end
 
+_M.vmethods = {
+    join = function(list, delim)
+        if delim == nil then
+            delim = ' '
+        end
+        local out = {}
+        for i = 1, #list, 1 do
+            table.insert(out, list[i])
+            if i ~= #list then
```

Comment:
We should avoid using `#` on tables since it will involve the `lj_tab_len` primitive which is a expensive `O(n)` operation, just like `table.insert`.